### PR TITLE
Update automated indicator enrichment to use parameters

### PR DESCRIPTION
**Pull request by smorrison-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [x ] Workflow verified as working in a test
- [ x] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [x ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ x] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

File: workflows/Automated_Indicator_Enrichment/help.md [hunk 1]
```diff
@@ -32,7 +32,13 @@ Sample Slack Trigger Commands:
 
 Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
 
-Once the workflow has been imported, **Update the first step with the channel name to suit your Slack environment!** by editing the input with the preset text of `change_me` to match the channel to monitor.
+This workflow leverages insightConnect's Parameters feature. This feature allows variables used multiple times throughout a workflow to be entered once and then referenced throughout the workflow.
```
Comment (mrinehart-r7) [inline workflows/Automated_Indicator_Enrichment/help.md:35]:
```suggestion
This workflow leverages InsightConnect's Parameters feature. This feature allows variables used multiple times throughout a workflow to be entered once and then referenced throughout the workflow. 
```

File: workflows/Automated_Indicator_Enrichment/help.md [hunk 2]
```diff
@@ -32,7 +32,13 @@ Sample Slack Trigger Commands:
 
 Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
 
-Once the workflow has been imported, **Update the first step with the channel name to suit your Slack environment!** by editing the input with the preset text of `change_me` to match the channel to monitor.
+This workflow leverages insightConnect's Parameters feature. This feature allows variables used multiple times throughout a workflow to be entered once and then referenced throughout the workflow. 
+
+There is one parameter you will need to configure in order to complete setup of your workflow:
+
+* `Slack Channel`:  the Slack channel name in your environment where the workflow should be triggered and respond
```
Comment (mrinehart-r7) [inline workflows/Automated_Indicator_Enrichment/help.md:39]:
```suggestion
* `Slack Channel`: The Slack channel name in your environment where the workflow should be triggered and respond
```

File: workflows/Automated_Indicator_Enrichment/help.md [hunk 3]
```diff
@@ -54,6 +60,7 @@ _There is no troubleshooting information at this time_
 
 # Version History
 
+* 1.2.0 - Leverage Parameters Feature | Update plugins
```
Comment (mrinehart-r7) [inline workflows/Automated_Indicator_Enrichment/help.md:63]:
```suggestion
* 1.2.0 - Leverage Parameters Feature | Update AbuseIPDB, VirusTotal, Whois, and Type Converter plugins
```
You'll also need to update the plugins table in the help.md to correspond to the version updates

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]

File: workflows/Automated_Indicator_Enrichment/help.md [hunk 4]
```diff
@@ -32,7 +32,13 @@ Sample Slack Trigger Commands:
...
-Once the workflow has been imported, **Update the first step with the channel name to suit your Slack environment!** by editing the input with the preset text of `change_me` to match the channel to monitor.
+This workflow leverages InsightConnect's Parameters feature. This feature allows variables used multiple times throughout a workflow to be entered once and then referenced throughout the workflow. 
+
+There is one parameter you will need to configure in order to complete setup of your workflow:
+
+* `Slack Channel`: The Slack channel name in your environment where the workflow should be triggered and respond
+
+To begin, select "Parameters" either from the Workflow Control Panel or from the Builder to begin configuration.
 
 After configuring the steps, activate the workflow in order to trigger it.
```
Comment (jnikitin-r7) [inline workflows/Automated_Indicator_Enrichment/help.md:44]: you need to edit the versions below, I think

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]
Need to update plugin versions in the Technical Details section

**Review by smorrison-r7 — COMMENTED** [review]
Updated Plugin Versions

File: workflows/Automated_Indicator_Enrichment/help.md [hunk 5]
```diff
@@ -48,19 +48,19 @@ Plugins utilized by workflow:
 
 |Plugin|Version|Count|
 |----|----|--------|
-|Type Converter|1.6.0|1|
-|AbuseIPDB|5.0.3|1|
+|Type Converter|1.7.0|
```
Comment (mrinehart-r7) [inline workflows/Automated_Indicator_Enrichment/help.md:51]:
```suggestion
|Type Converter|1.7.0|1|
```

File: workflows/Automated_Indicator_Enrichment/help.md [hunk 6]
```diff
@@ -48,19 +48,19 @@ Plugins utilized by workflow:
...
+|AbuseIPDB|5.0.6|
```
Comment (mrinehart-r7) [inline workflows/Automated_Indicator_Enrichment/help.md:52]:
```suggestion
|AbuseIPDB|5.0.6|1|
```

File: workflows/Automated_Indicator_Enrichment/help.md [hunk 7]
```diff
@@ -48,19 +48,19 @@ Plugins utilized by workflow:
...
 |IPStack|2.0.0|1|
-|VirusTotal|6.0.3|1|
-|WHOIS|2.0.2|1|
+|VirusTotal|6.0.4|
```
Comment (mrinehart-r7) [inline workflows/Automated_Indicator_Enrichment/help.md:54]:
```suggestion
|VirusTotal|6.0.4|1|
```

File: workflows/Automated_Indicator_Enrichment/help.md [hunk 8]
```diff
@@ -48,19 +48,19 @@ Plugins utilized by workflow:
...
+|WHOIS|3.0.1|
```
Comment (mrinehart-r7) [inline workflows/Automated_Indicator_Enrichment/help.md:55]:
```suggestion
|WHOIS|3.0.1|1|
```

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]

File: workflows/Automated_Indicator_Enrichment/workflow.spec.yaml [hunk 9]
```diff
@@ -3,13 +3,13 @@ products: ["insightconnect"]
 name: Automated_Indicator_Enrichment
 title: "Automated Indicator Enrichment"
 description: "Lookup indicators such as IP addresses or URLs using open source threat intelligence from Slack."
-version: 1.1.0
+version: 1.2.0
```
Comment (mrinehart-r7) [inline workflows/Automated_Indicator_Enrichment/workflow.spec.yaml:6]:
```suggestion
version: 2.0.0
```

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]

**Pull request merged**